src/main/java/software/amazon/awssdk/sar/ClientBuilderUsingRetryPolicy.java

```diff
@@ -0,0 +1,55 @@
+package software.amazon.awssdk.sar;
+
+import software.amazon.awssdk.awscore.retry.AwsRetryPolicy;
+import software.amazon.awssdk.core.retry.RetryMode;
+import software.amazon.awssdk.core.retry.RetryPolicy;
+import software.amazon.awssdk.core.retry.backoff.BackoffStrategy;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+/**
+ * Building clients using {@link RetryPolicy} is still supported and backwards compatible.
+ */
+public final class ClientBuilderUsingRetryPolicy {
+
+    /**
+     * Using RetryMode to build retry policies is still supported.
+     */
+    public static DynamoDbClient clientWithRetryPolicy01() {
+        return DynamoDbClient
+                .builder()
+                .overrideConfiguration(o -> o.retryPolicy(RetryMode.STANDARD))
+                .build();
+    }
+
+    /**
+     * Using concrete instances of retry policies is still supported.
+     */
+    public static DynamoDbClient clientWithRetryPolicy03() {
+        RetryPolicy policy = AwsRetryPolicy.forRetryMode(RetryMode.ADAPTIVE)
+                .toBuilder()
+                .numRetries(120)
+                .backoffStrategy(BackoffStrategy.none())
+                .build();
+
+        return DynamoDbClient
+                .builder()
+                .overrideConfiguration(o -> o.retryPolicy(policy))
```

Comment:
Can they set the retry strategy on the request? E.g. in the RequestOverrideConfiguration?

Reply:
Yes, we have an example about it using plugins.